fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/batch/RewriteJob.java

```diff
@@ -51,6 +53,10 @@ public RewriteJob(CascadesContext cascadesContext) {
                 .add(topDownBatch(ImmutableList.of(new FindHashConditionForJoin())))
                 .add(topDownBatch(ImmutableList.of(new PushPredicateThroughJoin())))
                 .add(topDownBatch(ImmutableList.of(new AggregateDisassemble())))
+                .add(topDownBatch(ImmutableList.of(new ColumnPruning())))
+                .add(topDownBatch(ImmutableList.of(new SwapFilterAndProject())))
+                .add(topDownBatch(ImmutableList.of(new MergeConsecutiveProjects())))
+                .add(topDownBatch(ImmutableList.of(new MergeConsecutiveFilters())))
```

Comment:
i think we need to add these three rules in one job

Reply:
done